diff --git a/node_modules/@docusaurus/core/lib/webpack/base.js b/node_modules/@docusaurus/core/lib/webpack/base.js
index c99cbef..b9d25fa 100644
--- a/node_modules/@docusaurus/core/lib/webpack/base.js
+++ b/node_modules/@docusaurus/core/lib/webpack/base.js
@@ -48,6 +48,7 @@ function getDocusaurusAliases() {
     return aliases;
 }
 exports.getDocusaurusAliases = getDocusaurusAliases;
+const cdnUrl = '//cdn.jsdelivr.net/gh/reactnativecn/react-native-website@gh-pages/';
 function createBaseConfig(props, isServer, minify = true) {
     var _a;
     const { outDir, siteDir, siteConfig, siteConfigPath, baseUrl, generatedFilesDir, routesPaths, siteMetadata, plugins, } = props;
@@ -93,7 +94,7 @@ function createBaseConfig(props, isServer, minify = true) {
             chunkFilename: isProd
                 ? 'assets/js/[name].[contenthash:8].js'
                 : '[name].js',
-            publicPath: baseUrl,
+            publicPath: isProd ? cdnUrl : baseUrl
         },
         // Don't throw warning when asset created is over 250kb
         performance: {
diff --git a/node_modules/@docusaurus/core/lib/webpack/client.js b/node_modules/@docusaurus/core/lib/webpack/client.js
index 586553d..6e8b122 100644
--- a/node_modules/@docusaurus/core/lib/webpack/client.js
+++ b/node_modules/@docusaurus/core/lib/webpack/client.js
@@ -32,7 +32,7 @@ function createClientConfig(props, minify = true) {
         optimization: {
             // Keep the runtime chunk separated to enable long term caching
             // https://twitter.com/wSokra/status/969679223278505985
-            runtimeChunk: true,
+            // runtimeChunk: true,
         },
         plugins: [
             new ChunkAssetPlugin_1.default(),
